x/crisis/keeper/keeper.go

```diff
@@ -85,9 +86,8 @@ func (k Keeper) AssertInvariants(ctx sdk.Context) {
 		}
 	}
 
-	end := time.Now()
-	diff := end.Sub(start)
-
+	diff := time.Since(start)
+	fmt.Println(">>>asserted all invariants", "duration", diff, "height", ctx.BlockHeight())
```

Comment:
This is a repetition (and prints messages to stdout), the following line already logs all the required information.

```suggestion
```